util/BUILD

```diff
@@ -57,3 +57,11 @@ py_binary(
         "//util/reggen:version",
     ],
 )
+
+py_test(
```

Comment:
Is there an autoformatting tool for this repo and/or BUILD files? I just eyeballed the formatting.

Reply:
Oh, nice! I think `:clang_format_check` should probably be a presubmit — it wants to make a bunch of changes in hw/ and sw/: ` 67 files changed, 157 insertions(+), 99 deletions(-)`.

I just added issue #13362 to track adding presubmit checks to infrastructure.